app/models/winning_bid.rb

```diff
@@ -0,0 +1,17 @@
+class WinningBid
```

Comment:
I'm not sure I understand why this needs to be a separate class since it's delegating for all of its functionality... Also, should it be in models since it isn't AR-backed?